nilearn/input_data/nifti_masker.py

```diff
@@ -199,8 +199,8 @@ def fit(self, imgs=None, y=None):
             self.affine_ = self.mask_img_.get_affine()
         # Load data in memory
         self.mask_img_.get_data()
-        if self.verbose > 10:
-            print "[%s.fit] Finished fit" % self.__class__.__name__
+        if self.verbose > 0:
```

Comment:
My recollection is that this is a print statement before immediately returning and that you want this only in some edge cases, for which you would set a crazy high verbosity. 

I seem to remember one of such edge cases was if you want to differentiate between the fit taking a long time and the next step in your analysis taking a long time (and not doing any printing when it starts for some stupid reason).